tools/workload-dependencies/workload-dependencies.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net9.0</TargetFramework>
```

Comment:
Can this be either `$(DotNetStableTargetFramework)` or `$(DotNetTargetFrameworkVersion)`?